test/tests/p5.SoundFile.js

```diff
@@ -1,142 +1,140 @@
-'use strict';
+var expect = chai.expect;
 
-  var expect = chai.expect;
+describe('p5.SoundFile', function () {
+  this.timeout(1000);
 
-  describe('p5.SoundFile', function () {
-    this.timeout(1000);
+  var sf;
+  var a = new p5.Amplitude();
```

Comment:
Needs to be wrapped in a `before` method

Reply:
(Make sure the tests pass in Firefox, it’s not running for me as-is in Firefox because of this)

Reply:
also no longer relevant